### PR TITLE
Bug42: Message too terse.

### PR DESCRIPTION
**Pull request by bjcoleman**
Made message more personal by referring to the reader.

### CODE REVIEW

**Comment by bjcoleman** [comment]
I really think this is an improvement to the code.

File: hi.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-hi there
+# making this more person
```
Comment (sommern) [inline hi.txt]: personable?

File: hi.txt [hunk 2]
```diff
@@ -1 +1,3 @@
...
+hi there, fine reader.
```
Comment (sommern) [inline hi.txt]: Might as well fix capitalization while we're at it.

**Comment by bjcoleman** [comment]
Better?

**Pull request merged**